action.yml

```
@@ -87,15 +87,17 @@ runs:
       with:
         ignore-empty-workdir: true
         enable-cache: false
-        version: "0.9.4"
+        version: "0.8.22"
     - name: Install Dependencies
       # Note tomli required for codespell with pyproject.toml
       env:
         INPUTS_SPELLING: ${{ inputs.spelling }}
       run: |
         # For debug
         # sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@main codespell tomli
-        packages="ultralytics-actions"
+        sudo env "PATH=$PATH" uv pip install --system --break-system-packages git+https://github.com/ultralytics/actions@fix-indent codespell tomli
+        # packages="ultralytics-actions"
+        packages=""
```

❗ **CRITICAL**: Avoid installing from a mutable feature branch and modifying the system interpreter. This introduces supply-chain risk, non-reproducibility, and can break the runner. Also, setting `packages=""` prevents `ultralytics-actions` from being installed later, which may cause runtime failures.

**Suggested change:**
```suggestion
        packages="ultralytics-actions"
```